docker-compose.yml

```
@@ -71,6 +84,13 @@ services:
   postgres:
     <<: *restart_policy
     image: "postgres:9.6"
+    healthcheck:
+      # Using default user "postgres" from sentry/sentry.conf.example.py or value of PING_USER if provided
+      test: ["CMD-SHELL", "pg_isready -U ${PING_USER:-postgres}"]
```

Let's scope the variable to postgres only:
```suggestion
      # Using default user "postgres" from sentry/sentry.conf.example.py or value of POSTGRES_USER if provided
      test: ["CMD-SHELL", "pg_isready -U ${POSTGRES_USER:-postgres}"]
```